language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -249,6 +249,18 @@ public static boolean isLangLib(String orgName, String moduleName) {
         return orgName.equals("ballerina") && moduleName.startsWith("lang.");
     }
 
+    /**
+     * Whether the given module is a langlib module or ballerina test module.
+     *
+     * @param moduleID Module ID to evaluate
+     * @return {@link Boolean} whether langlib or not ballerina test module
+     */
```

Comment:
```suggestion
    /**
     * Whether the given module is a langlib module or the Ballerina test module.
     *
     * @param moduleID Module ID to evaluate
     * @return {@link Boolean} <code>true</code> if the given module is either a langlib module or the Ballerina test module. <code>false</code> otherwise.
     */
```